canary/generator/utils.py

```diff
@@ -20,33 +37,58 @@ def process_file(filename):
         for measure in measures:
             measure_date = measure['date']
             hist_sum = sum(measure['histogram'])
-            values = (np.array(measure['histogram']))/float(hist_sum)
+            values = (np.array(measure['histogram'])) / float(hist_sum)
             if measure_date in series:
                 series['data'][measure_date] += np.array(values)
             else:
                 series['data'][measure_date] = np.array(values)
     return series
 
 
-def dict_to_df(data_dict):
-    buckets = data_dict['buckets']
-    data = data_dict['data']
+def X_dict_to_df(X_dict):
+    """
+    Parses X_dict do data frame
```

Comment:
nit: what's X_dict? Let's just briefly describe what it should be rather than referencing the variable name

Reply:
It's good you mention it because it was actually a mistake :)